### PR TITLE
Speakers None Field issue fixed

### PR DESCRIPTION
**Pull request by magdalenesuo**
Issue #2695
https://github.com/fossasia/open-event-orga-server/issues/2695

fixed:
https://cloud.githubusercontent.com/assets/20799954/21458543/6ce41b16-c95d-11e6-8fb7-680429c5accc.png

### CODE REVIEW

File: app/views/users/speakers.py [hunk 1]
```diff
@@ -41,6 +41,7 @@ def index_view(event_id):
 @event_speakers.route('/<int:speaker_id>/edit/', methods=('GET', 'POST'))
 def edit_view(event_id, speaker_id):
     speaker = get_speaker_or_throw(speaker_id)
+    print speaker
```
Comment (SaptakS) [inline app/views/users/speakers.py]: I think this line is not needed in the fix. Please remove this line.

Comment (SaptakS) [inline app/views/users/speakers.py]: Please remove this line. This log isn't required.

Comment (magdalenesuo) [inline app/views/users/speakers.py]: okay

File: app/templates/gentelella/admin/event/sessions/components/_speaker_component.html [hunk 2]
```diff
@@ -34,7 +34,7 @@
                           <input type="file" id="event-image-upload" name="event-image-upload"
                                  class="image-upload-btn no-js"
                                  accept="image/*"/>
-                          <label class="file-label" for="event-image-upload" id="event-image-upload-label">
+                          <label class="file-label" for="event-image-upload" id="event-image-upload-label" style="margin-top: 5vh">
```
Comment (SaptakS) [inline app/templates/gentelella/admin/event/sessions/components/_speaker_component.html]: I think 3vh is more suitable.

**Pull request merged**